src/js/components/validateData/ValidateDataFileComponent.jsx

```diff
@@ -151,7 +162,6 @@ export default class ValidateDataFileComponent extends React.Component {
 
             });
         }
-
         else if (item.missing_headers && item.missing_headers.length == 0 && item.duplicated_headers.length == 0 && item.error_type == 'header_errors') {
```

Comment:
wouldnt this make more sense to `else if` with the header error statement higher up in the page?